README.md

```diff
@@ -7,6 +7,8 @@ interacting with the configuration files and the results files from the
 ASAM Quality Checker Framework. With it, users can create their own
 Checker Bundles or Report Modules in Python.
 
+**Disclaimer**: The current version is the release candidate `1.0.0rc1`. The first official release is expected to be in November.
```

Comment:
Could be updated to `v1.0.0-rc1`, or just remove this. Then we don't need to update the rc number in the text when we have `-rc2`.

```suggestion
**Disclaimer**: The current version is a release candidate. The first official release is expected to be in November.
```